quickstart_template/301-jenkins-aptly-spinnaker-vmss.sh

```diff
@@ -159,10 +158,46 @@ run_util_script "jenkins/init-aptly-repo.sh" -vf "${vm_fqdn}" -rn "${repository_
 run_util_script "jenkins/add-aptly-build-job.sh" -al "${artifacts_location}" -st "${artifacts_location_sas_token}"
 
 echo "Setting up initial user..."
-echo "jenkins.model.Jenkins.instance.securityRealm.createAccount(\"$jenkins_username\", \"$jenkins_password\")"  > addUser.groovy
+
+# Using double quote for username and password would fail if contains dollar sign
```

Comment:
missing an "otherwise" in this comment?

Reply:
Actually here should use single quote. I changed the comment to eliminate ambiguity